lib/routes/account.js

```diff
@@ -1530,6 +1590,80 @@ module.exports = function (
     },
     {
       method: 'POST',
+      path: '/account/login/send_unblock_code',
+      config: {
+        validate: {
+          payload: {
+            email: validators.email().required()
+          }
+        }
+      },
+      handler: function (request, reply) {
```

Comment:
Probably needs to also check that unblock code is enabled, or else users can get emails but cannot act on them.